packages/next-codemod/bin/transform.ts

```diff
@@ -126,40 +145,77 @@ export async function runTransform(
 
   console.log(`Executing command: jscodeshift ${args.join(' ')}`)
 
-  const result = execa.sync(jscodeshiftExecutable, args, {
-    stdio: 'inherit',
-    stripFinalNewline: false,
+  const execaChildProcess = execa(jscodeshiftExecutable, args, {
+    // include ANSI color codes
+    // Note: execa merges env with existing env by default.
+    env: process.stdout.isTTY ? { FORCE_COLOR: 'true' } : {},
   })
 
-  if (result.failed) {
-    throw new Error(`jscodeshift exited with code ${result.exitCode}`)
-  }
+  // "\n" + "a\n" + "b\n"
+  let lastThreeLineBreaks = ''
+
+  if (execaChildProcess.stdout) {
+    execaChildProcess.stdout.pipe(process.stdout)
+    execaChildProcess.stderr.pipe(process.stderr)
+
+    // The last two lines contain the successful transformation count as "N ok".
+    // To save memory, we "slide the window" to keep only the last three line breaks.
+    // We save three line breaks because the EOL is always "\n".
+    execaChildProcess.stdout.on('data', (chunk) => {
+      lastThreeLineBreaks += chunk.toString('utf-8')
+
+      let cutoff = lastThreeLineBreaks.length
 
-  if (!dry && transformer === 'built-in-next-font') {
-    const { uninstallNextFont } = await prompts({
-      type: 'confirm',
-      name: 'uninstallNextFont',
-      message: 'Do you want to uninstall `@next/font`?',
-      initial: true,
+      // Note: the stdout ends with "\n".
+      // "foo\n" + "bar\n" + "baz\n" -> "\nbar\nbaz\n"
+      // "\n" + "foo\n" + "bar\n" -> "\nfoo\nbar\n"
+
+      for (let i = 0; i < 3; i++) {
+        cutoff = lastThreeLineBreaks.lastIndexOf('\n', cutoff) - 1
+      }
+
+      if (cutoff > 0 && cutoff < lastThreeLineBreaks.length) {
+        lastThreeLineBreaks = lastThreeLineBreaks.slice(cutoff + 1)
+      }
     })
+  }
+
+  try {
+    const result = await execaChildProcess
+
+    if (result.failed) {
+      throw new Error(`jscodeshift exited with code ${result.exitCode}`)
+    }
+  } catch (error) {
+    throw error
+  }
+
+  // With ANSI color codes, it will be "\x1B[39m\x1B[32m0 ok".
+  // Without, it will be "0 ok".
+  const targetOkLine = lastThreeLineBreaks.split('\n').at(-3)
+  const hasChanges =
+    targetOkLine.includes('ok') && !targetOkLine.includes('0 ok')
```

Comment:
Does this accidentally catch e.g. "10 ok"? We probably need to strip the ansi and do an exact match. We use `strip-ansi` in our tests